CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
+#### Added
+- Added new process filter for checking, if user has valid attribute 'isCesnetEligibleLastSeen'
+(is not older than 12 months) and redirect user to unauthorized page
```

Comment:
ADD: redirect the user to the unauthorized page in case the attribute has got invalid value (or is not set at all)